File: benchmarks/primitives/peers/int_mul.go
package main
import ("fmt"; "time")
func main(){ var n int64=300000000; var s uint64=1; t0:=time.Now()
 for i:=int64(0);i<n;i++{ s=s*6364136223846793005+1 }
 fmt.Printf("%d\nops: %d\nelapsed: %.6fs\n",s,n,time.Since(t0).Seconds()) }

File: benchmarks/primitives/peers/int_mul.rs
use std::time::Instant;
fn main(){ let n:i64=300000000; let mut s:u64=1; let t0=Instant::now();
 for _ in 0..n { s=s.wrapping_mul(6364136223846793005).wrapping_add(1); }
 let el=t0.elapsed().as_secs_f64();
 println!("{}\nops: {}\nelapsed: {:.6}s",s,n,el); }
